packages/portal/README.md

```diff
@@ -199,6 +199,61 @@ export default function Org({ variables }) {
 }
 ```
 
+#### i18n configuration
+
+Portaljs is configured by default to support both `English` and `french` subpath for language translation. But for subsequent users, this following steps can be use to configure i18n for other languages;
```

Comment:
@steveoni could you fix spelling of:

- "Portal.js" so we have `.`
- "French" so it's capitalized
- "can be used" so it is passive voice